src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -310,6 +343,44 @@ function EVMExecutor() {
           </Field>
         </>
       )}
+      {interactionType === TERMINAL_INDEX && (
+        <>
+          <Box
+            css={`
+              z-index: 1;
+            `}
+          >
+            <AceEditor
+              width="100%"
+              mode="jade"
+              value={code}
+              onChange={setCode}
+              fontSize={14}
+              showPrintMargin={false}
+              showGutter={false}
+              highlightActiveLine
+              setOptions={{
+                enableBasicAutocompletion: true,
+                enableLiveAutocompletion: true,
+                enableSnippets: true,
+                showLineNumbers: true,
+                tabSize: 2,
+              }}
+            />
+          </Box>
+          <Button
```

Comment:
we already have the button on line 401, let's just add another condition that when the interaction type is Terminal and the text inside the terminal is not null show it